tests/test_schemas.py

```diff
@@ -16,3 +17,25 @@ def test_mmd_xsd_strict(self):
         xsd_schema = os.path.join(pathlib.Path.cwd(), 'xsd', 'mmd_strict.xsd')
         xmlschema_mmd = ET.XMLSchema(ET.parse(xsd_schema))
         self.assertIsInstance(xmlschema_mmd, lxml.etree.XMLSchema)
+
+class TestXSLTs(unittest.TestCase):
+
+    def test_dcatap_xslt_instance(self):
+        transform_to_dcat = ET.XSLT(ET.parse('xslt/mmd-to-dcatap.xsl'))
+        self.assertIsInstance(transform_to_dcat, lxml.etree.XSLT)
+
+    def test_mmd_to_dcatap_translation(self):
+        inputpath = os.path.join(pathlib.Path.cwd(), 'input-examples', 'foo.xml')
+        dom = ET.parse(inputpath)
+        xsltfile = os.path.join(pathlib.Path.cwd(), 'xslt', 'mmd-to-dcatap.xsl')
+        xslt = ET.parse(xsltfile)
+        transform = ET.XSLT(xslt)
+        newdom = transform(dom)
+        self.graph = rdflib.Graph()
+        self.rdfObj = rdflib
```

Comment:
This line seems obsolete. If you need a variable in all the functions, it should be defined in a `setUp` function.
```suggestion
```